workbench/src/wb_monitor.h

```diff
@@ -22,7 +22,7 @@
 #include <glib.h>
 #include "wb_project.h"
 
-#if defined(HAVE_GIO) && GLIB_CHECK_VERSION (2, 46, 0)
+#if defined(HAVE_GIO)
```

Comment:
I don't think you need this `HAVE_GIO` stuff from the build system as GIO is part of GLib, Geany requires GIO already, and AFAIK some plugins too.

Reply:
Yes, Geany itself stopped checking back in [2011](https://github.com/geany/geany/commit/591417697010200441af6b6ed76ef28ef4731d83)

Reply:
Yes, geany itself is not using it, I will remove it. Maybe it should be removed from other plugins also.

Reply:
Done.